### DISCUSSION TITLE
"isElement" seems will be called in server-side

### DISCUSSION THREAD

**Discussion by KawaiiZapic** [element-plus/element-plus#6566] [General]
I caught a error in server-side rendering, "Element is not defined", according to the backtrace, I locate there is a call of isElement in Popover.
https://github.com/element-plus/element-plus/blob/dev/packages/components/popper/src/trigger.vue#L55  
That's pretty odd, isElement does not check the application is running in server or client.
@[JeremyWuuuuu](https://github.com/JeremyWuuuuu)

[the remark this answers is not in the document]
**Reply by jw-foss** [reply]
@KawaiiZapic This is indeed an error which seems like caused by me, I will fix it ASAP, either we should add this into `onMounted` block, or we should check `isClient`, would you mind creating an issue? Or I should trace that in this channel.

/cc @sxzz

[the remark this answers is not in the document]
**Reply by jw-foss** [reply]
@KawaiiZapic PR has been made in https://github.com/element-plus/element-plus/pull/6575
